### PR TITLE
Feat/lambda trigger

### PR DESCRIPTION
**Pull request by Durgaprasad-Budhwani**
CloudFront supports trigger for various purpose. Extended serverless component to support cloudfront triggers.

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -904,6 +928,39 @@ const removeDomainFromCloudFrontDistribution = async (cf, subdomain) => {
   }
 }
 
+const triggerTypes = ['viewer-request', 'origin-request', 'origin-response', 'viewer-response']
+
+const createLambda = async (lambda, trigger, subdomain, role) => {
+  const publishedLambda = await lambda({
+    runtime: 'nodejs10.x',
```
Comment (martrik) [inline utils.js:935]: Should we use `12.x`?

**Review by martrik — APPROVED** [review]
LGTM

File: utils.js (hunk 1 above)
Comment (Durgaprasad-Budhwani) [reply utils.js:935]: Good idea. Let me quickly make changes.

Comment (Durgaprasad-Budhwani) [reply utils.js:935]: I tried to publish but this is Lambda@Edge and it only supports following runtime environments nodejs8.10, nodejs10.x, nodejs6.10, python3.7

Comment (martrik) [reply utils.js:935]: ah sad!

**Pull request merged**